docs/guides/sdk/getting-started.md

```diff
@@ -0,0 +1,250 @@
+
+JavaScript
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItem } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(readItem('articles', 5));
+```
+
+TypeScript
+
+```ts
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItem } from '@directus/sdk@beta/rest/commands';
+
+type Schema = {
+	articles: { title: string; content: string };
+};
+
+const client = useDirectus<Schema>('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(readItem('articles', 5));
+```
+
+#### Read all items
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(readItems('articles'));
+```
+
+#### Read specific fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['id', 'title'],
+	})
+);
+```
+
+#### Read all fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*'],
+	})
+);
+```
+
+#### Read nested fields
+
+```js
+import { useDirectus } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/rest';
+import { readItems } from '@directus/sdk@beta/rest/commands';
+
+const client = useDirectus('http://directus.example.com')
+	.use(rest());
+
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*', { author: ['*'] }],
+	})
+);
+```
+
+### Using GraphQL
+
+Add the `graphql()` composable to the client, this enables the `.query(...)` method to query the collection.
+
+For example, to make a request to an `articles` collection.
+
+```ts
+import { useDirectus } from '@directus/sdk@beta';
+import { graphql } from '@directus/sdk@beta/graphql';
+
+type Article = {
+	title: string;
+	content: string;
+};
+
+type Schema = {
+	articles: Article;
+};
+
+const client = useDirectus<Schema>('http://directus.example.com')
+	.use(graphql());
+
+const result = await client.query<Article[]>(`
+    query {
+        articles {
+            id
+            title
+            content
+        }
+    }
+`);
+```
+
+> Feel free to customize the queries according to your project's GraphQL schema.
```

Comment:
Does this add anything to the example? any code example should be customized/adapted for your own project if copied.

Reply:
What do you mean?

Reply:
the comment is only about the quote itself